spacelift/data_azure_devops_integration.go

```diff
@@ -0,0 +1,63 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+)
+
+var azureDevopsIntegrationFields = struct {
+	OrganizationURL string
+	WebhookPassword string
```

Comment:
I think we may need also `Webhook endpoint` here, wdyt?
![image](https://user-images.githubusercontent.com/3693729/127996194-664bc736-f84c-46c5-bb41-5d5b28b5653f.png)

Reply:
![image](https://user-images.githubusercontent.com/63964167/127996513-aefb9de6-984f-4f4d-a310-96a7d0e34f4b.png)

Not available on graphql API + for no other VCS integration is webhook endpoint exposed.

Reply:
Fair enough. I am just curious, if this is used for some sort of automation we need a webhook address to provision at the VCS provider, and at the moment we can get it only through UI?

Reply:
Yeah, but it's statically generated. Deterministic.

Reply:
We should probably add it to the docs if it's not there.

Reply:
Since webhook endpoints are deterministic and not returned in any of the vcs providers at the moment, let's leave that as is. I'll bring it up on the next pgr to see if we need to update docs or maybe return webhook endpoints here (which I feel would be nice - having a complete data needed to provision an integration)